diff --git a/kernel/source/arch/mips/mips32r1/boot.S b/kernel/source/arch/mips/mips32r1/boot.S
index 005993d..0b55345 100755
--- a/kernel/source/arch/mips/mips32r1/boot.S
+++ b/kernel/source/arch/mips/mips32r1/boot.S
@@ -32,7 +32,6 @@
 #	.text
 	.align  4
 	.globl  _start
-	.ent    _start
 
 _start:
 	nop
@@ -87,6 +86,7 @@ _start:
 	li      t1, 0x2
 	lb      t2, 0(t0)
 	beq     t2, t1, 1f
+	nop
 
 	li      t0, 0xb8800220
 	lw      t1, 0(t0)
@@ -217,10 +217,8 @@ _start:
 	la      v0, main
 	jalr    v0
 	nop
-	.end    _start
 
 	.align  4
-	.ent    __usleep
 __usleep:
 	mfc0    v1,$9
 	HB
@@ -243,7 +241,6 @@ __usleep:
 3:
 	b       2b
 	addu    v0,a1,v0
-	.end
 
 	.set mips32
 
@@ -272,6 +269,5 @@ __usleep:
 */
 	.align  2
-	.ent    _invalidate_cache
 _invalidate_cache:
 #--- transition to kseg1 from undetermined kernel segment
 	la      t1, _inval_start_
@@ -383,7 +379,6 @@ _inval_d_:
 _inva_end_:
 	jr       ra
 	nop
-	.end _invalidate_cache
 
 #--------------------------------------------------------------------
 #
@@ -401,10 +396,8 @@ _inva_end_:
 	.byte   0
 	.align  7
 	.globl  _osGeneralExcept
-	.ent    _osGeneralExcept
 
 _osGeneralExcept:
 	la      k0, _osGeneralExceptHdl
 	j       k0
 	nop
-	.end    _osGeneralExcept
